### ISSUE TITLE
[SUGGESTION] Add artwork to badges

### ISSUE THREAD

**Issue by Matthias84** [Sav22999/common-voice-android#180]
**Description feedback/suggestion**

I like that you can unlock archivements to encourage people to contribute and start an friendly & sportif battle within the community. Other apps use visual badges to make this prizes a bit more appealing. For example streetcomplete (OpenStreetMap challenges):
![Screenshot_20210629-194629_Trebuchet](https://user-images.githubusercontent.com/217511/123866507-72efb580-d92d-11eb-868f-5ea6458ba582.png)
(see also here: https://github.com/streetcomplete/StreetComplete/tree/master/res/graphics/achievement%20graphics)

**Draft expected**
Visual feedback might make users more currious to finish challenges and to see new artworks

**Additional context**
You might want to ask communities of inkscape or at mastodon (["mastoart"](https://mastodon.social/tags/mastoart)) to contribute some GFX ressources?

**Comment by Sav22999** [comment]
Also the app has visual badges :smile:

Go to "Profile" > "Badges" (it's in the to-do list the showing of the badge when you unlock a new one)

**Comment by Matthias84** [comment]
Yes I know this badges. But how about to tweak them and give them a little polishment? Just some tuning to encourage contributors via gamification to check recording, contribute recordings, add profile information, ... and get a reward with a nice little graphic instead of only a text?

**Comment by Sav22999** [comment]
@Matthias84 the gamification is not the main goal of the app, but I'll think about and I'll create new badges with a better UI (maybe  you can try to ask to Inkscape, Krita, GIMP, Mastoart or other communities, id they want to contribute to the creation. I don't have any contacts with them, and I don't use mastodon neither)

Thanks for all these important suggestions and feedbacks
